packages/stylelint-plugin/src/kaizenTokens.ts

```diff
@@ -1,9 +1,11 @@
 import fs from "fs"
```

Comment:
The responsibilities of this file and `variableUtils` are becoming a little murky. I have a small idea about how it could be reworked (take it or leave it) which would be to combine all the `postcss-value-parser` helpers together. 🤷🏻‍♀️